gluejobutils/utils.py

```diff
@@ -1,5 +1,11 @@
 import json
 
+standard_date_format = 'yyyy-MM-dd'
+standard_date_format_python = '%Y-%m-%d'
+
+standard_datetime_format = 'yyyy-MM-dd HH:mm:ss'
```

Comment:
Are we sure we want this rather than `%Y-%m-%dT%H:%M:%S'`

I don't actually know what the difference is but if we're using it for partitions, then the space could possibly be an annoyance?

Reply:
I note that:
```
import datetime
datetime.datetime.now().isoformat()
> '2018-09-24T14:45:44.321695'
```

Reply:
agreed, turns out the 'T' breaks spark